pkg/cloud/aws/services/ec2/instances.go

```diff
@@ -309,6 +311,7 @@ func (s *Service) runInstance(i *v1alpha1.Instance) (*v1alpha1.Instance, error)
 
 	s.scope.EC2.WaitUntilInstanceRunning(&ec2.DescribeInstancesInput{InstanceIds: []*string{out.Instances[0].InstanceId}})
 
+	record.Eventf(s.scope.Cluster, "CreatedInstance", "Created new %s instance %q", role, *out.Instances[0].InstanceId)
```

Comment:
This will cause Machine-based instance creation/deletion (control plane, workers) to associate their events with the Cluster instead of the Machine.

To start, it might be a good idea to limit the events to the actuator-based methods, and then we can expand usage from there.

Reply:
Sounds good, the reason this was interesting is that it actually prints when a bastion host is spun up. I can change it to be limited to only machine-based ones instead and move the bastion creation event to the reconcile loop.